### PR TITLE
Ignore Bokeh UserWarning for multiple active properties in toolbar

### PR DESCRIPTION
**Pull request by Hoxbro**
Resolves https://github.com/holoviz/holoviews/issues/5869

The warnings we are seeing are the tools we are actually setting in: 

https://github.com/holoviz/holoviews/blob/6dcdb9aa4fe48be9df1110033e12ca83874fd3a1/holoviews/plotting/bokeh/element.py#L698-L705

We are using our merge_tools for merging tools instead of Bokeh. 

No special attention has been given to Bokeh 2, as it will be removed in https://github.com/holoviz/holoviews/pull/5891

### CODE REVIEW

File: holoviews/plotting/bokeh/plot.py [hunk 1]
```diff
@@ -586,13 +586,13 @@ def initialize_plot(self, ranges=None, plots=[]):
...
         plot = gridplot(plots[::-1],
-                        merge_tools=self.merge_tools,
+                        merge_tools=False,
                         sizing_mode=self.sizing_mode,
                         toolbar_location=self.toolbar)
         if self.sync_legends:
             sync_legends(plot)
         plot = self._make_axes(plot)
-        if bokeh3 and hasattr(plot, "toolbar"):
+        if bokeh3 and hasattr(plot, "toolbar") and self.merge_tools:
```
Comment (philippjfr) [inline holoviews/plotting/bokeh/plot.py:595]: Does that mean this was previously incorrect? Also what happens in the bokeh2 case now?

Comment (Hoxbro) [reply holoviews/plotting/bokeh/plot.py:595]: I think so. 

I have ignored Bokeh 2 as I plan to remove it soon with https://github.com/holoviz/holoviews/pull/5891. We don't even test for it anymore after the ImageStack PR was merged.

Comment (philippjfr) [reply holoviews/plotting/bokeh/plot.py:595]: Okay, sounds good.

**Review by philippjfr — APPROVED** [review]
Looks good, only just realized that the new `merge_tools` code is something I only implemented recently as part of the Bokeh 3.0 migration. The logic in there makes sense and matches our required semantics much better than what Bokeh does.

**Pull request merged**